Dockerfile-postgresql

```diff
@@ -5,11 +5,14 @@
 # the on-disk data in volumes. So the base image can not currently be upgraded
 # without users needing a manual pgdump and restore.
 
+# https://hub.docker.com/r/groonga/pgroonga/tags
+ARG PGROONGA_VERSION=2.3.4-alpine-14-slim
+FROM groonga/pgroonga:$PGROONGA_VERSION
```

Comment:
We should use `latest-alpine-14-slim`, or perhaps `latest-alpine-$POSTGRESQL_VERSION-slim`. There’s no reason to specify a particular PGroonga version.

Reply:
The reason I pinned PGroonga to a specific version is that you will get (more) deterministic builds with it. With `latest` you build a different image when build it after some time if there's a PGroonga upgrade, which might break sth.

But if you still like to `latest` I can push that change.

Reply:
@andersk ?

Reply:
Ping @andersk what do you think ?